apps/resources-wcm/src/main/webapp/pdf.js/locale/fr/viewer.properties

```diff
@@ -35,6 +35,7 @@ presentation_mode_label=Mode présentation
 open_file.title=Ouvrir le fichier
 open_file_label=Ouvrir le fichier
 print.title=Imprimer
+print.textContent= Imprimer
```

Comment:
Should we remove the spaces ?

Reply:
@rdenarie I added the space because i need it to be displayed

Reply:
These properties will not be translated using exoi10n but moz10l of pdf.js

Reply:
I'm afraid that modifications in Crowding will drop the space. Is there another way to add theses spaces ? 
How do it render without the space ?

Reply:
Without spaces ==>

![without_spaces](https://user-images.githubusercontent.com/70575401/149303852-5cdff3ea-36e9-4351-a914-361937c866bd.png)

With spaces ==>

![with_spaces](https://user-images.githubusercontent.com/70575401/149303948-460a1f71-6c0f-4ba2-9881-4282647cffa3.png)

Reply:
Is there a way to add a margin ? How it render without i18n ?

I'm pretty sure that the space will disappear when PO change crowding strings

Reply:
Ok i check

Reply:
Yes, i added a margin instead